elasticdl/python/collective_ops/communicator.py

```diff
@@ -57,15 +63,28 @@ def allreduce(self, data, op="MEAN"):
             )
             return CollectiveCommunicatorStatus.FAILED, data
         if self._ftlib is not None:
-            res = self._ftlib.wait_gradients_ready(data)
-            if res == FTAllReduceStatus.SUCCESS:
+            res = self._ftlib.wait_gradients_ready(params=data)
+            if (
+                res == CommLibStatus.SUCCESS
```

Comment:
This can be simplied to `res in [CommLibStatus.SUCCESS, FTAllReduceStatus.NO_NEED]`?